index.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <title>Fork Me! FCC: Test Suite Template</title>
+    <meta charset="UTF-8"  />
```

Comment:
i can see here, you don't have a viewport, i think you should add one, because your webhint linter error is coming from here, A 'viewport' meta element was not specified.